src/config.rs

```diff
@@ -79,6 +80,14 @@ impl Default for TermvizConfig {
                     g: 200,
                 },
             }],
+            marker_topics: vec![ListenerConfig {
```

Comment:
I think making the default config part of the listener module makes sense (same way id did for teleop) so the user of the config creates the default as well.